### PR TITLE
[BUGFIX] B/v1 509/remove df from result

### PR DESCRIPTION
**Pull request by billdirks**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/self_check/util.py [hunk 1]
```diff
@@ -29,12 +29,12 @@
     cast,
 )
 
-import numpy as np
 import pandas as pd
 from dateutil.parser import parse
 from typing_extensions import ParamSpec
 
 import great_expectations.compatibility.bigquery as BigQueryDialect
+import numpy as np
```
Comment (billdirks) [inline great_expectations/self_check/util.py]: This was ruff reordering imports.

File: great_expectations/constants.py [hunk 2]
```diff
@@ -0,0 +1 @@
+DATAFRAME_REPLACEMENT_STR = "<DATAFRAME>"
```
Comment (Shinnnyshinshin) [inline great_expectations/constants.py:1]: 👍🏼

File: great_expectations/datasource/fluent/batch_identifier_util.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from great_expectations.constants import DATAFRAME_REPLACEMENT_STR
+from great_expectations.core import IDDict
+
+
+def make_batch_identifier(identifer_dict: dict) -> IDDict:
+    batch_id: IDDict
+    if "dataframe" in identifer_dict:
+        batch_id = IDDict(identifer_dict, dataframe=DATAFRAME_REPLACEMENT_STR)
```
Comment (Shinnnyshinshin) [inline great_expectations/datasource/fluent/batch_identifier_util.py:8]: 🚀

**Review by Shinnnyshinshin — APPROVED** [review]
Thank you for the elegant solution @billdirks. LGTM :)

**Pull request merged**